CefSharp.OutOfProcess.Wpf.HwndHost/ChromiumWebBrowser.cs

```diff
@@ -279,13 +285,15 @@ public bool IsDisposed
         /// </summary>
         /// <param name="host">Out of process host</param>
         /// <param name="initialAddress">address to load initially</param>
-        public ChromiumWebBrowser(OutOfProcessHost host, string initialAddress = null)
+        /// <param name="requestContextPreferences">requestContextPreferences to set</param>
```

Comment:
The comment needs to be updated to describe the actual behaviour, a new `RequestContext` sharing the same settings as the `Global`.